script.js

```diff
@@ -0,0 +1,37 @@
+// Initializing an array for storing the quiz questions
+let quizQuestions = [];
+
+// Function to create an object from the quiz question
+document
+    .getElementById("quiz-form")
+    .addEventListener("submit", function (event) {
```

Comment:
Nice you have isolated your submit functionality into this event listener! I would consider abstracting this `function (event) {...` itself into it's own `onSubmit` function